### PR TITLE
272-cancel-order

### LINKED ISSUE #272: Cancel Order
Should display order info (you can use `Item` component) inside confirm modal
Should cancel it & show flash message on success

### PR DESCRIPTION
**Pull request by ololokiras**
#### What's this PR do?
cancel orders from active list
#### Where should the reviewer start?
src/views/ActiveOrders
#### How should this be manually tested?
remove some orders from account
#### Any background context you want to provide?
merge after https://github.com/bitshares/vuex-bitshares/pull/66
#### What are the relevant tickets?
closes #272 
#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/branch-272-cancel-order

### CODE REVIEW

File: src/views/ActiveOrders/ActiveOrdersTableItem.vue [hunk 1]
```diff
@@ -171,6 +171,9 @@ export default {
     },
     removeSize() {
       return this.expanded ? '14' : '10'
+    },
+    orderId() {
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrdersTableItem.vue]: ?

File: src/store/modules/activeOrders.js [hunk 2]
```diff
@@ -34,8 +34,9 @@ const actions = {
   changeSearchStr({ commit }, value) {
     commit(types.UPDATE_SEARCH_STR, value)
   },
-  removeActiveOrder({ commit, state }, ndx) {
-    commit(types.UPDATE_ACTIVE_LIST, state.activeList.filter((order, index) => ndx !== index))
+  async removeActiveOrder({ commit, state, dispatch }, orderId) {
```
Comment (roma219) [inline src/store/modules/activeOrders.js]: You can dispatch `transactions/cancelOrder` directly from component, this action is not needed

File: src/views/ActiveOrders/ActiveOrders.vue [hunk 3]
```diff
@@ -51,7 +53,8 @@ export default {
   },
   data() {
     return {
-      showRemoveModal: false
+      showRemoveModal: false,
+      orderForCancelIndex: {}
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrders.vue]: why index?

File: src/views/ActiveOrders/ActiveOrders.vue [hunk 4]
```diff
@@ -88,11 +91,30 @@ export default {
     }
   },
   methods: {
+    ...mapActions({
+      removeActiveOrder: 'activeOrders/removeActiveOrder'
+    }),
+    confirmRemove(order) {
+      console.log('remove item', order)
```
Comment (roma219) [inline src/views/ActiveOrders/ActiveOrders.vue]: remove console.log

**Review by roma219 — CHANGES_REQUESTED** [review]
- Make unlock popup appear before confirm
- Confirm button should display loader while order is being cancelled

File: src/views/ActiveOrders/ActiveOrders.vue (hunk 3 above)
Comment (ololokiras) [reply src/views/ActiveOrders/ActiveOrders.vue]: forgot rename(

**Comment by ololokiras** [comment]
fixed and updated

**Review by roma219 — APPROVED** [review]

**Pull request merged**